src/main/scala/mesosphere/marathon/tasks/TaskOpLogic.scala

```diff
@@ -0,0 +1,15 @@
+package mesosphere.marathon.tasks
+
+import mesosphere.marathon.core.task.Task
+import mesosphere.marathon.state.AppDefinition
+import org.apache.mesos.{ Protos => Mesos }
+
+/** Infers which TaskOps to crate for given app definitions and offers. */
+trait TaskOpLogic {
```

Comment:
Just thinking about the name, which is not very clear to me.
Isn't that the TaskOpFactory? (Where the other one is just a helper?)